.ci/jobs/observability-docs-mbp.yml

```diff
@@ -0,0 +1,43 @@
+---
+- job:
+    name: apm-server/observability-docs-mbp
```

Comment:
NIT:
I'm not sure if this is the default location for this particular project, maybe another folder?